src/hooks/useStrategies.tsx

```diff
@@ -100,6 +128,7 @@ export const useStrategiesWithFiat = (
 type StrategyCtx = ReturnType<typeof useStrategyFilter>;
 export const StrategyContext = createContext<StrategyCtx>({
   strategies: [],
+  filteredStrategies: [],
```

Comment:
I would rather rename to `allStrategies` & `strategies`

Reply:
I had it initially as this but I think calling `filteredStrategies` and `strategies` vs `strategies` and `allStrategies` respectively makes more sense as it not obvious when using the hook and reading the code that `strategies` is in fact filtered when allStrategies is not deconstructed from the hook's outputs:

```
const { strategies } = useStrategiesCtx() // NOT obvious that the strategies are filtered here
const { allStrategies } = useStrategiesCtx() // obvious that the strategies are NOT filtered here
```

vs


```
const { filteredStrategies } = useStrategiesCtx() // obvious that the strategies are filtered here
const { strategies } = useStrategiesCtx() // can be safely assumed that the strategies are NOT filtered here
```


Can change if you feel strongly about the first option